app/assets/javascripts/training/components/search_results.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const SearchResults = ({ slides }) => {
+  if (slides && slides.length > 0) {
+    return (
+      <ul className="training-libraries no-bullets no-margin action-card-text">
+        {slides.map((slide, index) => (
+          <li key={index}>
+            <a href={`https://dashboard.wikiedu.org/training/${slide.path}`} target="_blank">
```

Comment:
don't think this is correct? The root url will not always be `https://dashboard.wikiedu.org/`

Reply:
Hi, while I didn't make any changes on this and not very sure, I think it will always be that url because the slides controller that has the data is using the TrainingSlide model.